qds_sdk/cluster.py

```diff
@@ -216,6 +216,13 @@ def _parse_create_update(cls, args, action):
                              default=None,
                              help="Use hadoop1 instead of hadoop2. This is the default.")
 
+        spark = hadoop_group.add_mutually_exclusive_group()
+        spark.add_argument("--use-spark",
+                           dest="use_spark",
+                           action="store_true",
+                           default=None,
+                           help="Turn on spark for this cluster")
```

Comment:
Add a parameter which let's me edit a spark cluster to become a non spark cluster.